monty-31/src/x86_64_avx512/packing.rs

```diff
@@ -169,6 +231,44 @@ fn add<MPAVX512: MontyParametersAVX512>(lhs: __m512i, rhs: __m512i) -> __m512i {
 // [1] Modern Computer Arithmetic, Richard Brent and Paul Zimmermann, Cambridge University Press,
 //     2010, algorithm 2.7.
 
+/// Perform a partial Montgomery reduction on each 64 bit element.
+/// Input must lie in {0, ..., 2^32P}.
+/// The output will lie in {-P, ..., P} and be stored in the upper 32 bits.
+#[inline]
+#[must_use]
+fn partial_monty_red_unsigned_to_signed<MPAVX512: MontyParametersAVX512>(
+    input: __m512i,
+) -> __m512i {
+    unsafe {
+        // We throw a comfuse compiler here to prevent the compiler from
```

Comment:
Nit:
```suggestion
        // We throw a confuse compiler here to prevent the compiler from
```
This comment is copy-pasted several times below, and the fix should be applied to all of them.